qa-pipelines/pipeline-presets/single-brain.yml

```diff
@@ -0,0 +1 @@
+---
```

Comment:
is this needed?

Reply:
It's good practice to begin yml files to start with `---` which is the yml document separator (it allows multiple yml documents to exist in one file or stream). The QA pipeline deployment file concatenates several ymls when deploying pipelines (though I think just for values). It might not be necessary here (because I don't think other ymls get concatenated with the pipeline definition) but it does make it more future-proof.